src/diffusers/pipelines/text_to_video_synthesis/pipeline_text_to_video_zero.py

```diff
@@ -0,0 +1,521 @@
+from dataclasses import dataclass
+from typing import Callable, List, Optional, Union
+
+import numpy as np
+import PIL
+import torch
+import torchvision.transforms as T
```

Comment:
We can't import from `torchvision` Could you try to use native PyTorch here?

Reply:
Done!